doc/api/readline.md

```diff
@@ -318,24 +318,20 @@ changes:
     description: Symbol.asyncIterator support is no longer experimental.
 -->
 
-> Stability: 2 - Stable
-
 * Returns: {AsyncIterator}
 
 Create an `AsyncIterator` object that iterates through each line in the input
 stream as a string. This method allows asynchronous iteration of
-`readline.Interface` objects through `for`-`await`-`of` loops.
+`readline.Interface` objects through `for await...of` loops.
```

Comment:
There are twomore `` `for`-`await`-`of` `` at the end of the doc, in "Example: Read File Stream Line-by-Line" section.